webapp/components/admin_console/storage_settings.jsx

```diff
@@ -258,6 +262,43 @@ export default class StorageSettings extends AdminSettings {
                     onChange={this.handleChange}
                     disabled={!this.state.enableFileAttachments}
                 />
+                <BooleanSetting
+                    id='EnableClientSideEncryption'
+                    label={
+                        <FormattedMessage
+                            id='admin.file.enableClientSideEncryptionTitle'
+                            defaultMessage='Encrypt Files at the Mattermost server'
```

Comment:
@hmhealey  it is set to only show if the S3 driver is selected.  It should also work with S3 compatible systems like Minio.